src/exception/call_stack.cr

```diff
@@ -167,7 +175,7 @@ struct Exception::CallStack
         next if @@skip.includes?(file)
 
         # Turn to relative to the current dir, if possible
-        file = Path.new(file).relative_to(Process::INITIAL_PWD)
+        file = file.lchop(CURRENT_DIR)
```

Comment:
Could you explain what was the problem with this? I feel like I don't understand what's the original problem and how this solves it.

Reply:
This is your code... And I described the problem as clear as I could, so I'm quite puzzled as to what don't you understand...

Reply:
Sorry, what do you mean it's my code? If I wrote it, maybe it happens years ago so I don't remember what this is all about. I guess someone changed it to use `relative_to` and that was wrong?

Reply:
Yes, you wrote it, and I'm sorry you don't remember what is it about (even though you left the comments about it).

And the issue is described in #10169 if you cared to read the original description...

Reply:
And not "someone" changed it but @straight-shoota - who acknowledged and approved this PR already...

Reply:
Yes, the issue is described there, but the specs don't show that it was solved. You just removed some slashes from the beginning of the string to match, making the tests actually weaker.

Reply:
Wrong, the specs are fixed since they were noop before - I'd be happy if you'd care to check out beforehand.

Reply:
Also I've added spec covering this exact case...

Reply:
Okay. But I feel like you are fighting and I'm trying to understand. In any case, I merged it because I think it's fine and also because @straight-shoota approved it. Sorry if I caused any trouble.

Reply:
Yeah, I do felt that:

- you don't care to read/understand the linked issue
- your comments regarding validity of my changes were written in an authoritative mannee, yet they were unfounded and unchecked
- logic presented in the comments was way off - questioning reverts, your own code and others...

Reply:
You are right. The reason is that I took just 20 minutes off my time to go and review and merge PRs. I shouldn't have reviewed this one because it would consume a lot more time than that for me.

Reply:
FWIW: I felt your language @Sija became a bit agressive without a clear indication why. I still can't follow why you felt not recognized by @asterite when he just asked to understand the meaning of this change.
Yes, technically it's all described, and I immediately picket up on what's wrong and how you're fixing it. But it's not very easy to follow what's going on when you're not familiar with the code (even if you're the original author a few years back!).